assets/src/edit-story/components/tablist/tablistPanel.js

```diff
@@ -82,7 +84,11 @@ const TablistPanel = ({
         </Badge>
       </TabButton>
       <Scrollable maxHeight={maxHeight}>
-        <TabPanel aria-labelledby={`${title}-${panelId}`} role="tabpanel">
+        <TabPanel
+          id={panelId}
+          aria-labelledby={`${title}-${panelId}`}
+          role="tabpanel"
+        >
```

Comment:
fix aria stuff (aria-controls was referencing nothing)

Reply:
Should `id` be made a required prop instead of adding a fallback `backupId` that will again cause this to reference nothing?

Reply:
> Should `id` be made a required prop instead of adding a fallback `backupId` that will again cause this to reference nothing?

```js
const backupId = useMemo(() => uuidv4(), []);
const panelId = id || backupId;
```

I added `backupId` here in case an `id` wasn't passed in. This way, the button will be tied to the expandable content with `aria-controls={panelid}` since `panelId` will always be defined.

Another way to do that would be to make `id` a required prop - and if that's preferred I can change that 👍